examples/inference_server/server.py

```diff
@@ -0,0 +1,99 @@
+"""Flask inference server.
+
+Implements a Flask server that handles inference requests for some input via a HTTP handler.
+To run the server, run the following command from the root directory:
+`FLASK_APP=examples/inference_server/server.py flask run`
+"""
+
+import os
+import random
+import re
+import string
+import subprocess
+
+from flask import Flask, request, abort
+from werkzeug.utils import secure_filename
+import sky
+
+from inference import INFERENCE_RESULT_MARKER
+
+LOCAL_UPLOAD_FOLDER = '/Users/isaac/Dropbox/Berkeley/Sky/sky/examples/inference_server/uploads'
+REMOTE_UPLOAD_FOLDER = '/remote/path/to/folder'
```

Comment:
Can we make this example easier to run? It would be nice if users can run this example without any modification to the code.